doc/provisioning.md

```diff
@@ -84,28 +89,27 @@ At the time of writing, the currently implemented provider-specific placement di
   - Availability Zone: both the AWS and OpenStack providers support `zone=<zone>`, directing the provisioner to start an instance in the specified availability zone.
   - MAAS: `<hostname>` directs the MAAS provider to acquire the node with the specified hostname.
 
-Availability Zone Spread
-------------------------
+## Availability Zones
 
-For Juju providers that know about Availability Zones, instances will be 
-automatically spread across the healthy availability zones to maximise service 
+For Juju providers that know about Availability Zones, instances will be
+automatically spread across the healthy availability zones to maximise service
```

Comment:
No need to capitalize "availability zones".

Reply:
Done, see 217f424ef0.